docs/development/index.rst

```diff
@@ -0,0 +1,21 @@
+=======================
+Developer documentation
+=======================
+
+.. toctree::
+  :maxdepth: 2
+  
+  Developer API <devapi>
+  Writing plugins <plugins>
+
+
+Developer Installation
+----------------------
+
+For developers, we provide a simple mechanism to allow importing
+imageio from the cloned repository <https://github.com/imageio/imageio.git>.
+See the file ``imageio.proxy.py`` for details.
```

Comment:
... I think we should advocate using `pip install -e .` instead. That proxy thing is from before pip existed.

Reply:
Oh wow, I knew imageIO is rather old, but I never realized it comes from the pre-pip era. I agree that this should be updated.

Reply:
Not 100% sure if imageio is, but that proxy file is ;)

Reply:
Done :)